datafusion/common/src/scalar.rs

```diff
@@ -77,8 +77,8 @@ pub enum ScalarValue {
     /// large binary
     LargeBinary(Option<Vec<u8>>),
     /// list of nested ScalarValue
-    List(Option<Vec<ScalarValue>>, Box<DataType>),
-    /// Date stored as a signed 32bit int days since UNIX epoch 1970-01-01
+    List(Option<Vec<ScalarValue>>, Box<Field>),
+    /// Date stored as a signed 32bit int
```

Comment:
```suggestion
    /// Date stored as a signed 32bit int days since UNIX epoch 1970-01-01
```

Seems to have gotten lost in the merge